subcommittees/mentoring/minutes/minutes-2017-08-28.md

```diff
@@ -0,0 +1,276 @@
+
+- Juan: I think maby a commitee partly in "flux" working on mini projects. Doing reflecting planning cycles...  Secondly, there are normal standard agenda items that will be part of a status quo.
+What my experience is thus far is that the mentoring commitee facilitates / assist new members to join and get involved with the community as they provide opportunity to become more involved in an organic way
+- Tobin: as a well-advertised place that instructors can go to for help. I had a lot of local support, but i can see it being hard for trainers who don't have more experience geographically close.
+- Jamie: MC is a place where instructors can turn to for feedback, help, and to share problems/issues they may be facing as an instructor - MC hears these issues and works together to create solutions i.e. through Juan's suggestion of "mini projects" like the onboarding doc
+- Toby: a friendly group of people interested in making becoming/being an instructor easier and more enjoyable. It should be clear that this is a subcommittee that anyone can join, and meetings that people should feel free to drop in on, without any expectation that they will keep returning forever i.e. attendance does not equal/require "membership". 
+  - (PS are we likely to change the name? It's a bit strange to me that we don't (actively) do much mentoring. This almost stopped me from coming to the meetings originally - I didn't/don't feel experienced enough to be mentoring others!)
+
+###### Second Meeting
+
+- ML: In order to build sustainability in a group - rotations are needed! Have parties (or some form of recognition) to take a moment to acknoweldge the service of our community members. 
+- EB: We can have the admin take the information 
+- UPDATE TO ONBOARDING DOCUMENT: Send the names of everyone who was present in the discussion (including host and notetaker)
+
+- Plan:
+  - add a slot to new demo sessions for a designated notetaker under instructor training checkout slots
+  - remove notetaker role and have a designated notetaker (or 2) in the participants
+
+- Steps:
+  - schedule an additional discussion session each week (that doesn't conflict with anything else on the Carpentries calendar)
+  - put those events on the Carpentries calendar
+  - put those events on the Etherpad
+  - put out a call for hosts (also contact existing notetakers and ask them to change from notetaking to hosting a different session)
+  - advertise to Discuss/Twitter that we now have new sessions
+- MS: will communicate with Jamie/Mateusz about a plan 
+
+
+#### *Walk-ons: please add additional topics below
+
+- MK: we should contribute to the __new carpentries repo__. I'd like to start to do this. anyone else want to help?
+  - TH: +1
+  - CK: +1
+  - JH: +1
+
+- PA: __new community for Latin America__. Translating lessons to Spanish, hosting spanish language discussion sessions etc
+  - MK: Raniere is already hosting a few sessions in spanish, I think
```

Comment:
I meant Rayna (not Raniere), but @rgaiacs may have been hosting those too :)

Reply:
Rayna and Sue McClatchy are hosting demo sessions in Spanish.

Reply:
Rayna and Sue McClatchy are hosting demo sessions in Spanish.